### PR TITLE
Update roadmap.tpl

### PR DESCRIPTION
**Pull request by landrok**
Clean closing bracket

### CODE REVIEW

**Comment by peterdd** [comment]
about showall toggle: See FS#1968 https://bugs.flyspray.org/1968

similiar to the other pure HTML/CSS toggles like advanced search form toggle.

**Comment by landrok** [comment]
Nice idea, good inspiration but rewriting of roadmap.tpl js is not the topic here.

**Comment by peterdd** [comment]
But this patch changes behavior of allTasks-toggles too.. (?)

**Comment by landrok** [comment]
Sorry my branch did not contain JS inline code anymore. I'll pull a new pull in a few moments.

**Comment by peterdd** [comment]
'allTasks' still missing.

class=task_milestone$milestone_id needed for what?

Well, do not invest to much here for moving JS out here. Can all be done by CSS in future.


I haven't invested much into roadmap.tpl yet. I think this pages power is limited for user.

Could be made much more powerful and eyecandy with the now available css-based task attributes styling like showing tasktypes as icons, progress milestone bars with seperated percent sections (finished bugs,features, undone bugfixes, undone feature)

**Comment by landrok** [comment]
https://github.com/Flyspray/flyspray/commit/63cfd8a7bba257758705d6d0ac1811d5ed854f39

ok... but work is done.

class=task_milestone$milestone_id is for sibling without php/js meltings

File: themes/CleanFS/templates/roadmap.tpl [hunk 1]
```diff
@@ -23,17 +23,17 @@ function showAll(allTasks)
...
+-</script>
 
 <div class="box roadmap">
 <h3><?php echo Filters::noXSS(L('roadmapfor')); ?> <?php echo Filters::noXSS($milestone['name']); ?>
 
     <?php if (count($milestone['open_tasks'])): ?>
     <small class="DoNotPrint">
-      <a href="javascript:showAll(allTasks<?php echo Filters::noXSS($milestone['id']); ?>)"><?php echo Filters::noXSS(L('expandall')); ?></a> |
-      <a href="javascript:hideAll(allTasks<?php echo Filters::noXSS($milestone['id']); ?>)"><?php echo Filters::noXSS(L('collapseall')); ?></a>
+      <a href="javascript:showAll(<?php echo Filters::noXSS($milestone['id']); ?>)"><?php echo Filters::noXSS(L('expandall')); ?></a> |
```
Comment (peterdd) [inline themes/CleanFS/templates/roadmap.tpl]: @landrok: i mean here.

**Pull request closed without merging**